packages/mongodb-memory-server-core/src/util/DryMongoBinary.ts

```diff
@@ -24,6 +26,7 @@ export interface DryMongoBinaryOptions extends BaseDryMongoBinaryOptions {
 export interface DryMongoBinaryNameOptions {
   version: NonNullable<DryMongoBinaryOptions['version']>;
   arch: NonNullable<DryMongoBinaryOptions['arch']>;
+  platform: NonNullable<DryMongoBinaryOptions['version']>;
```

Comment:
why exactly was this added? i currently dont quite see where this used that it would be required to be in here

Reply:
It is required in the DryMongoBinary's getBinaryName, `await new MongoBinaryDownloadUrl(opts).getArchiveName()`.